js/options/options.js

```diff
@@ -395,12 +396,22 @@ let Options = (function(){
             let countries = SyncedStorage.get("regional_countries");
             countries.forEach(country => {
                 HTML.beforeBegin(addAnotherWrapper, generateRegionSelect(country));
+                addAnotherWrapper.previousSibling.querySelector(".custom-link__close").addEventListener("click", e => {
+                    let select = e.target.closest(".country_parent").querySelector(".regional_country");
+                    select.value = "";
+                    saveOption("regional_countries");
```

Comment:
This is the same listener as 10 lines below

Reply:
Abstracted it